README.md

```diff
@@ -160,13 +160,14 @@ See unit test in xt folder for more examples
     youtubeAnalytics : v1,v1beta1 : http://developers.google.com/youtube/analytics/
     youtubereporting : v1 : https://developers.google.com/youtube/reporting/v1/reports/
 
-# AUTHOR
+# AUTHORS
 
-Pavel Serikov <pavelsr@cpan.org>
+- Pavel Serikov <pavelsr@cpan.org>
```

Comment:
contributors are now automatically generated by dist zilla as noted in previous pull request.